backends/qualcomm/runtime/backends/QnnProfiler.cpp

```diff
@@ -20,7 +20,7 @@ QnnProfile::QnnProfile(
   if (profile_level != QnnExecuTorchProfileLevel::kProfileOff) {
     const QnnInterface& qnn_interface = implementation_.GetQnnInterface();
     Qnn_ErrorHandle_t error = qnn_interface.qnn_profile_create(
-        backend_->GetHandle(), static_cast<int>(profile_level), &handle_);
```

Comment:
Do you want to fixed profile_level?

Reply:
maybe we should add the level to `QnnExecuTorchProfileLevel`. Maybe typo? I saw `kProfileOptrace = 3` there.

@shewu-quic , how about the linting profile level? Should we reserve a number for that?

Reply:
Sorry, somehow the code change exported is out of sync with the internal diff. Just re-exported; the profile level should be 2 for both kProfileDetailed (2) and kProfileOptrace (3).

Reply:
Updated to remove the hardcoded value of QNN profile level.